### DISCUSSION TITLE
Edit on Github Button on every page

### DISCUSSION THREAD

**Discussion by JMNSTYLZ** [requarks/wiki#5816] [Help / Questions] [answered]
Hi All,

When i look at the Requarks wikijs docs, I see on every page the button with: 'edit on github'.
I want this too on my own wikijs instance for my company. We host the files on a internal gitlab. Is this also possible to show?

When I look at the admin there is no option to enable this. 

Would love to hear from you!

**Comment by JMNSTYLZ** [comment, marked as answer]
can be closed.

**Reply by NGPixel** [reply]
Adding the answer here in case someone finds this question:

It can be enabled in the Administration > General section.
